refinery/data_set_manager/utils.py

```diff
@@ -9,22 +9,19 @@
 import time
 import urlparse
 
+import core
```

Comment:
I can look into customizing the behavior of isort if it really matters, but my sense is that we want some consistent order, but which particular order it is doesn't matter so much?